hivemq-edge/src/frontend/src/components/react-icons/hm/HmInput.tsx

```diff
@@ -0,0 +1,29 @@
+import { GenIcon, IconBaseProps } from 'react-icons'
+
+export const HmInput = (props: IconBaseProps) =>
+  GenIcon({
```

Comment:
isn't there a way to add an actual SVG file instead of using `GenIcon` with a JSON representation of an SVG?
for example with something like [vite-plugin-svgr](https://www.npmjs.com/package/vite-plugin-svgr)?

Reply:
There is indeed, and it's used for images. For icons, I just decided to keep the same formalism as the `react-icon` library used